README.md

```diff
@@ -1,24 +1,52 @@
-# README
+# Daily Expenses
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Daily Expenses is a budget management application where users can register and use the features of the application. Users can create categories, assign the categories a particular icon, and add expenses to the categories. Users can also see the total amount spent on each category.
 
-Things you may want to cover:
+![screenshot](./gif.gif)
 
-* Ruby version
+[Live Link](https://yourdailyexpenses.herokuapp.com/)
```

Comment:
- [ ] Good job including the live link as suggested by the previous reviewer, but some of the pages are not working properly. Please kindly make sure all pages work fine. You can also take the option of asking users to include an image URL as this is for development purposes.

Reply:
@AnselemOdims, I already stated the problem with Heroku and the active storage that rails provide. I also provided the link from heroku that describes that in order to make active storage to work we need amazon s3 but i can't use that since to it needs credit card to sign up.